draft-ietf-taps-interface.md

```diff
@@ -1642,22 +1653,14 @@ Passive Connections can also be added to the same group -- e.g., when a Listener
 receives a new Connection that is just a new stream of an already active multi-streaming
 protocol instance.
 
-Changing one of the Connection Properties on one Connection in the group
-changes it for all others. Message Properties, however, are not
-entangled. For example, changing `Timeout for aborting Connection` (see
-{{conn-timeout}}) on one Connection in a group will automatically change this
-Connection Property for all Connections in the group in the same way. However,
-changing `Lifetime` (see {{msg-lifetime}}) of a Message will only affect a
-single Message on a single Connection, entangled or not.
-
 If the underlying protocol supports multi-streaming, it is natural to use this
-functionality to implement Clone. In that case, entangled Connections are
+functionality to implement Clone. In that case, Connections in a Connection Group are
 multiplexed together, giving them similar treatment not only inside endpoints,
 but also across the end-to-end Internet path.
 
 Note that calling Clone() can result in on-the-wire signaling, e.g., to open a new
-connection, depending on the underlying Protocol Stack. When Clone() leads to
-multiple connections being opened instead of multi-streaming,
+transport connection, depending on the underlying Protocol Stack. When Clone() leads to
+multiple such connections being opened instead of multi-streaming,
```

Comment:
/multiple such connections being opened/the opening of multiple such connections,/